pyaedt/modeler/circuits/PrimitivesNexxim.py

```diff
@@ -865,6 +865,55 @@ def create_voltage_pulse(self, compname=None, value_lists=[], location=[], angle
 
         return cmpid
 
+    @pyaedt_function_handler()
+    def create_voltage_pwl(
+        self, compname=None, time_list=[0], voltage_list=[0], location=[], angle=0, use_instance_id_netlist=False
+    ):
+        """Create a pwl voltage source.
+
+        Parameters
+        ----------
+        compname : str, optional
+            Name of the voltage pulse. The default is ``None``.
+        time_list : list, optional
+            List of time points for the pwl voltage source. The default is ``[0]``.
+        voltage_list : list, optional
+            List of voltages for the pwl voltage source. The default is ``[0]``.
+        location : list of float, optional
+            Position on the X axis and Y axis.
```

Comment:
While I prefer capitalizing X and Y as you have done, in this library (and virtually all libraries), a lower-case x and y are almost always used.